helm/net-exporter/values.yaml

```diff
@@ -15,7 +15,7 @@ timeout: 5s
 
 image:
   registry: quay.io
-  name: giantswarm/net-exporter
+  repository: giantswarm/net-exporter
```

Comment:
for consistency

Reply:
you're actually reverting change @rossf7 made via https://github.com/giantswarm/net-exporter/commit/0e6081d60c43a0ea26c54e7571c8a50bd8864030

I think direction where Ross was going is that whichever is not currently using net-exporter as app, either TC or CP, that it should, and then it gets these registry values from appcatalog configmap. I understand there's now kind of clash between installations/draughtsman config and appcatalog configuration.

Reply:
@calvix Yes the value was renamed from `repository` to `name` because its simpler and sorter.

See https://github.com/giantswarm/fmt/blob/master/helm/image_templating.md

Otherwise this looks fine. Making the templating work with both value structures is the right call IMO. 👍